src/manifolds/FlagOrthogonal.jl

```diff
@@ -0,0 +1,132 @@
+
+function check_vector(
```

Comment:
Can we document this when a vector is valid? Maybe write a doctoring for a `is_vector`?

Reply:
Sure, though as far as I can see we document `check_vector`, not `is_vector`.

Reply:
Oh, well, that is also fine, then.